packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.constants.js

```diff
@@ -1,7 +1,17 @@
 export const VERSION_ENUM_KEY = "models['cloud.kube.VersionEnum'].enum";
 export const ANTI_AFFINITY_MAX_NODES = 5;
+export const SCALE_DEFAULT_VALUES = {
```

Comment:
I don't see this constant used anywhere...

Reply:
No it's related to another topics that i worked on for kube part.
Once it was merged i got a conflicts on this file which i resolved.

Reply:
Thanks for observation